src/tests/functions/exp/test_exp.py

```diff
@@ -39,6 +41,28 @@ def test_exp(random_mock, client_mock):
     assert ("", 200) == result
 
 
+def get_timeout_timestamp_value(param):  # pragma: no cover
+    if param == "last_message_timestamp":
+        return (datetime.now() - timedelta(seconds=1)).strftime(DATETIME_FORMAT)
+
+    return 0
+
+
+@patch(f"{MODULE_PATH}.Client")
+def test_exp_timeout(client_mock):
+
+    request_mock = MagicMock()
+    request_mock.get_json.return_value = GOOD_BODY
+
+    client_mock.return_value.collection.return_value.document.return_value.collection.return_value.document.return_value.get.return_value.get.side_effect = (
```

Comment:
this.is.way.too.long.like.for.real.maybe.separate.dot.with.new.line?

Reply:
`black` doesn't accept it, it will remove the `\` used for the new line
But yeah, we should find an other solution